### ISSUE TITLE
Missing some documentation on how to make work multiple domains, different ports on the same docker node

### ISSUE THREAD

**Issue by williamdes** [traefik/traefik#7605]
This is a documentation request, here is the working example I did ([Source](https://gist.github.com/ismailyenigul/867132f298664156068b6e85847c0dfe#gistcomment-3545998))

### `docker-compose.yml`

```yml
version: "3"
services:
   traefik:
    image: traefik:v2.2
   # More config ...
  gophish:
    image: gophish/gophish
    volumes:
      - ${GOPHISH_DB_DIR}:/app/database
      - ${GOPHISH_STATIC_DIR}:/app/static/endpoint
      - ${GOPHISH_STATIC_DIR}:/opt/gophish/static/endpoint
    restart: on-failure:5
    dns_search: example.fr
    hostname: www.example.fr
    domainname: example.fr
    container_name: gophish
    networks:
      internal-server:
        aliases:
          - example.fr
          - www.example.fr
    environment:
      - ADMIN_LISTEN_URL=0.0.0.0:3333
      - ADMIN_USE_TLS=false
      - PHISH_LISTEN_URL=0.0.0.0:888
      - PHISH_USE_TLS=false
      - GOPHISH_INITIAL_ADMIN_PASSWORD=public
      - DB_FILE_PATH=/app/database/gophish.db
    labels:
      traefik.enable: true
      traefik.docker.network: internal-server

      # The HTTPS config
      traefik.http.routers.www-router.rule: host(`www.example.fr`)
      traefik.http.routers.www-router.entrypoints: websecure
      traefik.http.routers.www-router.tls: "true"
      traefik.http.routers.www-router.tls.certresolver: "acmeresolver"
      traefik.http.routers.www-router.service: www

      # The HTTP config
      traefik.http.routers.www-router-http.rule: host(`www.example.fr`)
      traefik.http.routers.www-router-http.entrypoints: web
      traefik.http.routers.www-router-http.service: www

      # The port config for the 2 routers above config
      traefik.http.services.www.loadbalancer.server.port: 888

      # The HTTPS config
      traefik.http.routers.admin-router.rule: host(`gophish-admin.example.fr`)
      traefik.http.routers.admin-router.entrypoints: websecure
      traefik.http.routers.admin-router.tls: "true"
      traefik.http.routers.admin-router.tls.certresolver: "acmeresolver"
      traefik.http.routers.admin-router.service: administration

      # The HTTP config
      traefik.http.routers.admin-router-http.rule: host(`gophish-admin.example.fr`)
      traefik.http.routers.admin-router-http.entrypoints: web
      traefik.http.routers.admin-router-http.service: administration

      # The port config for the 2 routers above config
      traefik.http.services.administration.loadbalancer.server.port: 3333

networks:
  internal-server:
    driver: bridge

```

### `traefik.yml`
If you see on the dashboard a router Traefik did create itself.
```yml
providers:
    docker:
        exposedbydefault: false
        defaultRule: "HostRegexp(`{{ index .Labels \"com.docker.compose.service\"}}-defaultRule.localhost`)"
```

### ACME resolver by http (`traefik.yml`)
```yml

certificatesResolvers:
    acmeresolver:
        acme:
            email: tech@example.fr
            storage: acme.json
            httpChallenge:
                entryPoint: web

**Comment by williamdes** [comment]
Is this fixed now that the PR is merged?

**Comment by ldez** [comment]
Yes fixed by #7610
